src/modules/Painter/tools/blackline.ts

```diff
@@ -0,0 +1,31 @@
+import Tool from '@/modules/Tool';
+import {ActionType} from '@/constants';
+
+export default Tool.create({
```

Comment:
I think it's really good idea to provide interface for making custom tool!!
but also It would be nice to have a basic set of various tools for those who don't know html canvas well

Reply:
Yes, i'm planning various basic tools.
also, i think this will help the development of boardy by becoming a new boardy ecosystem.